python/eos/analysis.py

```diff
@@ -202,6 +203,11 @@ def _perplexity(weights):
             perplexity = np.exp(entropy) / len(normalized_weights)
             return perplexity
 
+    @staticmethod
+    def _ess(weights):
+        """Helper function that computes the effective sample size of an array of weights"""
+        from pypmc.tools.convergence import ess
```

Comment:
I meant: you could simply use
```python
return pypmc.tools.convergence.ess(weights)
```
which would be the full name of the ``ess`` function. Importing it might take a performance penalty, but I'm not sure.

Reply:
This require importing the entire pypmc in the helper function which I wanted to avoid. For the mooment it is imported only in `sample_pmc`